### ISSUE TITLE
Recent updates?

### ISSUE THREAD

**Issue by hrvoje459** [exul/matrix-rocketchat#82]
Hi,

is there any news or updates regarding development of this service? From the screenshot in the readme it looks like it was able to facilitate messaging users directly from matrix to rocketchat (other way around?) which is the what we are looking for in our use case. Only other alternative for rocketchat that I could find was matterbridge but it only bridges rooms/channels together. Do you have any plans on developing this bridge further or do you have any recommendations on how to proceed with bridging matrix and rocketchat together?

**Comment by exul** [comment]
Hi,

There are no news regarding the development of this service. I did start a re-write of it, but didn't make significant progress due to time constraints. A lot of the components that were used in the version that you can see in the repository are outdated because it was early days for both matrix on rust. I'm not yet sure how I'll continue, if I'll ever publish a re-write or not, that's why there is a comment in the Readme that the project is currently not maintained.
Regarding alternatives, I'm not aware of any, the page on matrix.org only shows this bridge: https://matrix.org/bridges/#rocketchat

**Comment by MurzNN** [comment]
Maybe now will be easier to reuse some library for bridging like mx-puppet-bridge (or alternative on other language), instead of doing all from scratch?

**Comment by exul** [comment]
Yes, sure it would be easier, but I didn't just build the bridge to have a bridge for Rocket.Chat, I also wanted to learn something. It was my first project where I used Rust and because there were no high level libraries I also learned a lot about Matrix. Even when sticking with Rust there is a lot of progress since I started this project ([ruma](https://github.com/ruma/ruma) as well as as [matrix-rust-sdk](https://github.com/matrix-org/matrix-rust-sdk). At the end of the day it's still a huge task to develop and maintain a bridge. So without dedicating a lot of time, it doesn't make sense to start over, which is why I'm not making any promises that I will (but also not saying that I won't ;-)).
